projects/plugins/jetpack/modules/notes.php

```diff
@@ -192,12 +192,18 @@ public function admin_bar_menu() {
 		$third_party_cookie_check_iframe = '<span style="display:none;"><iframe class="jetpack-notes-cookie-check" src="https://widgets.wp.com/3rd-party-cookie-check/index.html"></iframe></span>';
 
 		$title = self::get_notes_markup();
+
+		// The default fallback is `en_US`. Remove underscore if present, noting that lang codes can be more than three chars.
+		if ( str_contains( '_' ) ) {
```

Comment:
Do we need that `if` condition at all?

Reply:
Heh, nope. It was an incomplete leftover when I was testing a less-efficient method. :^)